retrofit-adapters/rxjava/src/main/java/retrofit/ObservableCallAdapterFactory.java

```diff
@@ -0,0 +1,167 @@
+package retrofit;
+
+import java.io.IOException;
+import java.lang.reflect.ParameterizedType;
+import java.lang.reflect.Type;
+import rx.Observable;
+import rx.Subscriber;
+import rx.functions.Action0;
+import rx.functions.Func1;
+import rx.subscriptions.Subscriptions;
+
+/**
+ * TODO docs
```

Comment:
Yeah, definitely need this. I think the thing that's important to mention is that each subscription yields a single value, computed fresh for each subscriber.